medicines/web/infrastructure/README.md

```diff
@@ -0,0 +1,36 @@
+# products.mhra.gov.uk Website Infrastucture
+
+Instructions on getting the website set up.
+As steps are completed, the developer completing those steps is responsible for updating this document.
+This document should become a guide for setting up the website infrastructure on a new environment and for connecting to the existing environment.
+
+## Steps
+
+1. Docker time!
+
+   Create a `Dockerfile` in `/medicines/web` which will run `yarn dev`, serving up the Nextjs website.
+   Verify that this is working locally by running the docker image, and connecting to the website.
+
+   You will need to push the docker image to the container registry for each of our environments.
+
+2. Create Kubernetes pod
+
+   First, you'll need YAML specifying the pod's configuration.
```

Comment:
Any example doc?

Reply:
Best example we have would be the one for the API at `../../api/infrastucture/<environment>/deployment.yml`.
I can update to include this guidance.

Reply:
Sounds good, or copy/paste it inline with values removed.